tests/e2e/test_driver.py

```diff
@@ -341,7 +395,7 @@ def test_escape_single_quotes(self):
             assert rows[0]["col_1"] == "you're"
 
             # Test escape syntax in parameter
-            cursor.execute("SELECT * FROM {} WHERE {}.col_1 LIKE :var".format(table_name, table_name), parameters={"var": "you're"})
+            cursor.execute("SELECT * FROM {} WHERE {}.col_1 LIKE %(var)s".format(table_name, table_name), parameters={"var": "you're"})
```

Comment:
I erroneously changed this test case while developing the SQLAlchemy dialect. This merely reverts it to the correct syntax.

Reply:
If a server supports native param query and the app sets use_inline_params False, it'd use `SELECT * FROM {} WHERE {}.col_1 LIKE :var` format?

Reply:
This is an old test. Prior to this PR, it was the _only_ e2e test of inline parameter rendering. We could consolidate it into the same test file with the other tests and update it to dynamically shift its syntax. For now, if we ran this test with `use_inline_params=False`, it would fail with a syntax error. 

However, since we're running the e2e test suite with `use_inline_params=True` by default the test is technically correct. And the new tests added in this PR already add coverage of this behaviour.

Also, we have unit tests for the parameter escaper which I wrote last year.